illumio.json

```diff
@@ -605,7 +645,10 @@
                         "/orgs/1/sec_policy/1221"
                     ],
                     "column_name": "Href",
-                    "column_order": 0
+                    "column_order": 0,
+                    "contains": [
+                        "provision href"
```

Comment:
After the discussion, I understand this implementation. Two things:

- We should probably prefix these, e.g. `illumio workload href` to avoid potential name conflicts
- This `provision href` should be replaced with a list of all supported types (`illumio rule set href`, `illumio virtual service href`, ...)

Reply:
Changed